routers/api/v1/repo/issue_subscription.go

```diff
@@ -200,22 +200,13 @@ func GetIssueSubscribers(ctx *context.APIContext, form api.User) {
 		return
 	}
 
-	iw, err := models.GetIssueWatchers(issue.ID)
+	iwl, err := models.GetIssueWatchers(issue.ID)
```

Comment:
How about just change `models.GetIssueWatchers`'s return from `[]*IssueWatch` to `IssueWatchList`?

Reply:
done